### PR TITLE
compute pending for all non refunded epochs

### PR DESCRIPTION
**Pull request by mwamedacen**
closes FRNT-158

### CODE REVIEW

**Review by samthomson — APPROVED** [review]
good you got to this :+1:

File: scripts/gas-refund-program/persistance/db-persistance.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export async function fetchTotalRefundedPSP(
       ...(toEpoch ? { epoch: { [Op.lt]: toEpoch } } : {}),
     },
     dataType: 'string',
```
Comment (Velenir) [inline scripts/gas-refund-program/persistance/db-persistance.ts:55]: `dataType` overrides the resulting type?
And of course it's not type properly in the lib?
Of course

**Review by Velenir — APPROVED** [review]
Looks good

File: scripts/gas-refund-program/persistance/db-persistance.ts (hunk 1 above)
Comment (mwamedacen) [reply scripts/gas-refund-program/persistance/db-persistance.ts:55]: problem with types yes

**Pull request merged**